testplan/testing/multitest/driver/app.py

```diff
@@ -341,12 +323,11 @@ def starting(self):
                 self._proc = None
             raise
 
-    def started_check(self, timeout=None):
+    def started_check(self, timeout: Union[int, None] = None) -> None:
```

Comment:
Union[int, None] or Optional[int] ?

Reply:
Oh right, missed that return statement, updating.